src/services/member/controller.ts

```diff
@@ -103,12 +104,44 @@ const controller: FastifyPluginAsync = async (fastify) => {
   );
 
   // delete member
+  /**
+   * @deprecated
+   * TODO: Fix this endpoint as it should not need the member ID.
```

Comment:
I'm just highlighting this `TODO`. I think it should be converted as an issue ?

Reply:
Yes done !